pkg/online/online.go

```diff
@@ -185,38 +186,11 @@ func CreateAppFromOnline(pendingApp *types.PendingApp, upstreamURI string, isAut
 
 	if isAutomated && kotsKinds.Config != nil {
 		// bypass the config screen if no configuration is required
-		licenseSpec, err := kotsKinds.Marshal("kots.io", "v1beta1", "License")
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to marshal license spec")
-		}
-
-		configSpec, err := kotsKinds.Marshal("kots.io", "v1beta1", "Config")
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to marshal config spec")
-		}
-
-		configValuesSpec, err := kotsKinds.Marshal("kots.io", "v1beta1", "ConfigValues")
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to marshal configvalues spec")
-		}
-
-		identityConfigSpec, err := kotsKinds.Marshal("kots.io", "v1beta1", "IdentityConfig")
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to marshal identityconfig spec")
-		}
-
-		configOptions := kotsadmconfig.ConfigOptions{
-			ConfigSpec:         configSpec,
-			ConfigValuesSpec:   configValuesSpec,
-			LicenseSpec:        licenseSpec,
-			IdentityConfigSpec: identityConfigSpec,
-			// TODO: are there ever registry settings here?
-		}
-		needsConfig, err := kotsadmconfig.NeedsConfiguration(configOptions)
+		registrySettings := registrytypes.RegistrySettings{} // TODO: are there ever registry settings here?
```

Comment:
`store.GetStore().GetRegistryDetailsForApp(appID)` will return this.  If registry info is used in a template, in a required config item, it will be blank without registry info.